### PR TITLE
chore: add automation for common templates release-v0.31

### PR DESCRIPTION
**Pull request by ksimon1**
chore: add automation for common templates release-v0.31

### CODE REVIEW

File: ci-operator/config/kubevirt/common-templates/kubevirt-common-templates-release-v0.31.yaml [hunk 1]
```diff
@@ -0,0 +1,392 @@
+base_images:
+  base:
+    name: "4.17"
```
Comment (akrejcir) [inline ci-operator/config/kubevirt/common-templates/kubevirt-common-templates-release-v0.31.yaml:3]: These versions are newer than in `kubevirt-common-templates-master.yaml`.
Can you post another PR for `master` branch?

**Review by akrejcir — COMMENTED** [review]
Looks good.

/lgtm

File: ci-operator/config/kubevirt/common-templates/kubevirt-common-templates-release-v0.31.yaml (hunk 1 above)
Comment (ksimon1) [reply ci-operator/config/kubevirt/common-templates/kubevirt-common-templates-release-v0.31.yaml:3]: Yes, I wanted to do it as follow up PR

Comment (ksimon1) [reply ci-operator/config/kubevirt/common-templates/kubevirt-common-templates-release-v0.31.yaml:3]: https://github.com/openshift/release/pull/57029

**Pull request merged**